_posts/2017-09-26-iviewer-0-3-0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Release of OMERO.iviewer 0.3.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.iviewer 0.3.0
+---
+Today we are releasing our version 0.3.0 of OMERO.iviewer: an OMERO.web app
+for visualizing images in OMERO. This version requires OMERO 5.3.3 or newer.
+
+This 0.3.0 version is still work-in-progress and some areas are likely to be
+modified.
+
+This version includes:
+
+- support for saving the image in the viewport as a .png file
+- support for exporting basic ROIs measurement (area and length) in a CSV file
```

Comment:
maybe something like "... exporting basic measurements (area and length) of ROIs in ..."?